### PR TITLE
vendor: bump crlfmt a78e1c207bc0 to b3eff0b87c79

### PR DESCRIPTION
**Pull request by stevendanna**
This bumps the version of cockroachdb/crlfmt, pulling in the following
changes:

  - add -srcdir option similar to goimports
  - use printer options that match gofmt when simplifying
  - Fix link to CockroachDB Go coding guidelines
  - using go module to manage dependencies
  - add simplify from gofmt -s
  - Update README with usage info
  - set default tab width to 2

Additionally, this pulls in a new version of golang.org/x/tools:

   golang/tools@d8d6ddb...v0.1.0

A manual entry for golang.org/x/sys has been added to the bazel
WORKSPACE to avoid an older version from being pulled in during `make
bazel-generate`.

Release note: None

### CODE REVIEW

**Comment by irfansharif** [comment]
Just checked it out locally (and rebasing off of `master` to pick up a recent bazel build breakage), this SHA ends up tripping up the bazel build (try `bazel build --color=yes @cockroach//pkg/cmd/cockroach-short --sandbox_debug`). I'm pretty sure this isn't the diff that's at fault, so looking now.

**Comment by irfansharif** [comment]
Hm, I'm still not really sure why we're seeing this error now, but it's because of code changes in `x/sys` that we're now picking up. Specifically https://github.com/golang/sys/blob/22da62e12c0cd9c1da93581e1113ca4d82a5be14/execabs/execabs.go#L41-L43.

I think we'll have to update how we use `stringer` in the bazel sandbox, evidently some behaviour has changed that we were implicitly relying on. Essentially all instances of `org_golang_x_tools//cmd/stringer:stringer` in our codebase. I won't be able to investigate this week, but I know @rickystewart, @alan-mas, and others in #dev-inf are working around bazel as of late. So perhaps they'd be able to help.

**Comment by stevendanna** [comment]
I can also take a look between onboarding sessions this week. Apologies that this didn't build out of the box, I had assumed that CI was running the bazel builds.

**Comment by irfansharif** [comment]
Nothing to apologize for! The "bazel build" is currently optional in CI (and I think was recently suspended because of other issues). I'll defer to Ricky here, but if I'm reading correctly that this PR isn't immediately urgent, I'd personally prefer to try and address the /x/sys incompatibility first before merging (as opposed to breaking the bazel build, which a small minority of us try to use day-to-day).

**Comment by stevendanna** [comment]
> if I'm reading correctly that this PR isn't immediately urgent

Yup, not urgent at all

**Comment by stevendanna** [comment]
Added a commit with a "quick fix" for the bazel build problem.

**Comment by rickystewart** [comment]
The "quick fix" seems fine to me.

**Comment by stevendanna** [comment]
Rebased this on master and I think it should be good to go.

File: build/STRINGER.bzl
Comment (irfansharif) [inline build/STRINGER.bzl:9]: Mind adding a brief comment explaining why this was necessary?

**Review by irfansharif — APPROVED** [review]

**Comment by stevendanna** [comment]
bors r=irfansharif

**Pull request merged**
